src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -883,6 +889,55 @@ def detect(
                 how='diagonal',
             )
 
+    def compute_event_properties(
+            self,
+            event_properties: str | tuple[str, dict[str, Any]]
+            | list[str | tuple[str, dict[str, Any]]],
+            name: str | None = None,
+            verbose: bool = True,
+    ) -> pm.EventDataFrame:
+        """Calculate an event property for and add it as a column to the event dataframe.
+
+        Parameters
+        ----------
+        event_properties: str | tuple[str, dict[str, Any]] | list[str | tuple[str, dict[str, Any]]]
+            The event properties to compute.
+        name: str | None
+            Process only events that match the name. (default: None)
+        verbose: bool
+            If ``True``, print information about the progress. (default: True)
+        Raises
+        ------
+        InvalidProperty
+            If ``property_name`` is not a valid property. See
+            :py:mod:`pymovements.events.event_properties` for an overview of supported properties.
+        RuntimeError
+            If specified event name ``name`` is missing from ``events``.
+            If no events are available to compute event properties. Consider calling detect before.
+
+        Returns
+        -------
+        pm.EventDataFrame
+            Returns self, useful for method cascading.
+        """
+        if self.events is None:
+            raise RuntimeError(
+                'No events available to compute event properties. '
+                'Consider calling detect before.',
+            )
+
+        if verbose is True:
```

Comment:
To log the debug message one would have to set `verbose` to `True` and set the logging level to `DEBUG`.

So probably the verbose parameter isn't needed after all in this method and the if condition can be updated to
 ```python
if logger.isEnabledFor(logging.DEBUG):
```
to avoid creating the debug string if not needed

Reply:
the verbose thing was there just to be consistent with the Dataset method, but maybe we can remove it together with the logging code above and keep it simple;
the message is not very informative either as it shows the same thing over and over again when the method is called from the Dataset